coredns/README.md

```diff
@@ -0,0 +1,68 @@
+# CoreDNS Integration
+
+## Overview
+Get metrics from CoreDNS in real time to:
+
+* Visualize and monitor DNS failures and cache hits/misses
+## Setup
+
+### Installation
+1. Download the Datadog Agent.
+2. Download the `datadog_checks/coredns/coredns.py` file.
+3. Place it in the Agent's checks.d directory.
+
+### Configuration
+1. Download the `datadog_checks/coredns/data/conf.yaml.example`
+2. Place file in the Datadog Agent's `conf.d` directory.
+3. Rename file to coredns.yaml
+4. Edit file to point to your server and port.
+5. [Restart the Agent][3] to begin sending CoreDNS metrics to Datadog.
+
+### Validation
+
```

Comment:
Validation section should follow the same layout as all other integrations in integrations core:
https://github.com/DataDog/integrations-core/tree/master/apache#validation

We don't display the result since it differs between agent v5 and v6.